### DISCUSSION TITLE
Static IP

### DISCUSSION THREAD

**Discussion by mtkhayes3** [Security-Onion-Solutions/securityonion#1900] [General] [answered]
Installed SO 2.3.2 only Manager Node so far using SO ISO and using Static IP but when system reboots it is using DHCP IP and I do not have access to the Web.  If I change IP in network-scripts I still cannot access the Web.

**Comment by mtkhayes3** [comment]
I used air gapped and it keep the static ip

**Comment by weslambert** [comment, marked as answer]
Thanks for reporting.  I haven't run into this myself, but will look into it.
